.github/workflows/clang-linux-test-x64.yml

```diff
@@ -0,0 +1,125 @@
+name: AscEmu CI (clang)
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+
+  Classic:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Environment
+      run: |
+        sudo apt-get update && sudo apt-get install -y zlib1g-dev
+        sudo apt-get update && sudo apt-get install -y libssl-dev
+        sudo apt-get update && sudo apt-get install -y libpcre3-dev
+        sudo apt-get update && sudo apt-get install -y libbz2-dev
+        sudo apt-get update && sudo apt-get install -y default-libmysqlclient-dev
+        sudo apt-get update && sudo apt-get install -y libmysql++-dev
+    - name: AscEmu Build Classic
+      run: |
+        gcc --version
+        cmake --version
+        mkdir ascemu_build_сlassic
+        cd ascemu_build_сlassic
+        cmake ../ -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -DCMAKE_BUILD_TYPE=Release -DBUILD_WITH_WARNINGS=0 -DBUILD_TOOLS=1 -DASCEMU_VERSION=Classic
+        cd ..
+        cd ascemu_build_сlassic
+        make -j4
+        sudo make install
+  TBC:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Environment
+      run: |
+        sudo apt-get update && sudo apt-get install -y zlib1g-dev
+        sudo apt-get update && sudo apt-get install -y libssl-dev
+        sudo apt-get update && sudo apt-get install -y libpcre3-dev
+        sudo apt-get update && sudo apt-get install -y libbz2-dev
+        sudo apt-get update && sudo apt-get install -y default-libmysqlclient-dev
+        sudo apt-get update && sudo apt-get install -y libmysql++-dev
+    - name: AscEmu Build TBC
+      run: |
+        gcc --version
+        cmake --version
+        mkdir ascemu_build_tbc
+        cd ascemu_build_tbc
+        cmake ../ -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -DCMAKE_BUILD_TYPE=Release -DBUILD_WITH_WARNINGS=0 -DBUILD_TOOLS=1 -DASCEMU_VERSION=TBC
+        cd ..
+        cd ascemu_build_tbc
+        make -j4
+        sudo make install
+  WotLK:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Environment
+      run: |
+        sudo apt-get update && sudo apt-get install -y zlib1g-dev
+        sudo apt-get update && sudo apt-get install -y libssl-dev
+        sudo apt-get update && sudo apt-get install -y libpcre3-dev
+        sudo apt-get update && sudo apt-get install -y libbz2-dev
+        sudo apt-get update && sudo apt-get install -y default-libmysqlclient-dev
+        sudo apt-get update && sudo apt-get install -y libmysql++-dev
+    - name: AscEmu Build WotLK
+      run: |
+        gcc --version
```

Comment:
Since we are using Clang, use clang --version

Reply:
up